docs/releases/v2.1.0.md

```diff
@@ -0,0 +1,24 @@
+# AWS Add-on Module Release 2.1.0
+
+Welcome to the latest release of the `aws` module for the [`Kubernetes Fury Distribution`](https://github.com/sighupio/fury-distribution) maintained by team SIGHUP.
+
+This is a minor release adding support for Kubernets `v1.24.x`.
+
+## Component Images 🚢
+
+| Component                  | Supported Version                                                                               | Previous Version |
+| -------------------------- | ----------------------------------------------------------------------------------------------- | ---------------- |
+| `cluster-austoscaler`      | [`v1.24.0`](https://github.com/kubernetes/autoscaler/releases/tag/cluster-autoscaler-1.24.0)    | `1.23.1`         |
```

Comment:
Maybe here there is the wrong version

Reply:
do you mean 1.23.1? Is in the "previous version" column

Reply:
ah right ✅ , i'm stupid